UI/MainScreen.cpp

```diff
@@ -1070,6 +1070,11 @@ UI::EventReturn MainScreen::OnForums(UI::EventParams &e) {
 UI::EventReturn MainScreen::OnExit(UI::EventParams &e) {
 	System_SendMessage("event", "exitprogram");
 
+#ifdef IOS
+	// This is being called before System_SendMessage("finish", "") because calling it afterwards causes a crash :/
+	void SpringBoardAnimatedExit();
+	SpringBoardAnimatedExit();
```

Comment:
This belongs here:
https://github.com/hrydgard/ppsspp/blob/c2f4fad56aada331bbd8dd7db99037285bab853f/ios/main.mm#L36

Currently that's `exit(0)`ing, it's not crashing or anything.  You can just put `[[UIApplication sharedApplication] animatedExit];` in there no?

-[Unknown]